aspnetcore/performance/caching/memory/sample/WebCache/Controllers/HomeController.cs

```diff
@@ -46,6 +46,33 @@ public IActionResult CacheTryGetValueSet()
     }
     #endregion
 
+    #region snippetSize
+    public IActionResult CacheTryGetValueSetSize()
+    {
+        DateTime cacheEntry;
+
+        // Look for cache key.
+        if (!_cache.TryGetValue(CacheKeys.Entry, out cacheEntry))
+        {
+            // Key not in cache, so get data.
+            cacheEntry = DateTime.Now;
+
+            // Set cache options.
+            var cacheEntryOptions = new MemoryCacheEntryOptions()
+                .SetSize(1024*2)
```

Comment:
Why are we adding a sample setting the size in WebCache? The memory cache in DI for this sample doesn't have a size limit configured so the sizes set here will just be ignored.